src/index.tsx

```diff
@@ -49,16 +49,30 @@ export class PDFObject extends React.PureComponent<Props> {
     forcePDFJS: false,
     assumptionMode: true,
   };
+  public constructor(props: Props) {
```

Comment:
if use arrow function to the `embed`, no needs this constructor

Reply:
@sugarshin  done